docs/development/new-packages.md

```diff
@@ -41,22 +40,44 @@ To determine if a package has C extensions, check if its `setup.py` contains
 any compilation commands.
 ```
 
-### 2. Creating the `meta.yaml` file
+## Building a Python package
+
+### 1. Creating the `meta.yaml` file
+
+To build a Python package, you need to create a `meta.yaml` file that defines a
+"recipe" which includes build commands and "patches" (source code edits),
+amongst other things.
 
 If your package is on PyPI, the easiest place to start is with the
-{ref}`mkpkg tool <pyodide-mkpkg>`. From the Pyodide root directory, install the
-tool with `pip install ./pyodide-build`, then run:
+{ref}`mkpkg tool <pyodide-mkpkg>`. First clone and build the Pyodide git repo
+like this:
+
+```bash
+git clone https://github.com/pyodide/pyodide
+cd pyodide
+make
```

Comment:
You don't need to build Python to use `mkpkg`. But I guess if it's a binary package, you'll need to build it later. Soon it shouldn't be necessary to build a binary package at all =)

Reply:
If it's required sometimes, then probably best to leave it like this? Can be removed once it's not necessary?